API.yaml

```diff
@@ -474,12 +511,18 @@ paths:
         required: true
         schema:
           type: string
-      - name: queryFilterID
-        in: query
-        description: Filter ID previously created
-        schema:
-          type: integer
-          format: int32
+      requestBody:
```

Comment:
You cannot use requestBody for any GET. The Swagger web editor (not vs-code plug-in) will mark it as an error.